.github/workflows/patrol-integration-test.yaml

```diff
@@ -0,0 +1,56 @@
+name: Integration tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  JAVA_VERSION: 17
+  FLUTTER_VERSION: 3.22.2
+
+jobs:
+  mobile_integration_test:
+    permissions:
+      contents: "read"
+      id-token: "write"
+
+    name: Run integration tests for mobile apps
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ngrok
+      cancel-in-progress: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Authenticate to Google Cloud
+        uses: "google-github-actions/auth@v2"
+        with:
+          project_id: ${{ secrets.GOOGLE_CLOUD_PROJECT_ID }}
+          workload_identity_provider: ${{ secrets.GOOGLE_CLOUD_WORKLOAD_IDENTITY_PROVIDER_ID }}
+          service_account: ${{ secrets.GOOGLE_CLOUD_SERVICE_ACCOUNT }}
```

Comment:
any risk if we use the same Firebase project with our FCM? @tk-nguyen

Reply:
No risk. In fact this is how we're using it right now.